Runtime/Scripts/InstanceRenderingInfo.cs

```diff
@@ -154,10 +156,10 @@ private Matrix4x4 worldMatrix
         {
             get
             {
-                if (gameObject == null)
+                if (instanceParent == null)
                     return Matrix4x4.identity;
-                    
-                return gameObject.transform.localToWorldMatrix;
+
+                return instanceParent.transform.localToWorldMatrix;
```

Comment:
Since `instanceParent` is already a Transform object, we don't need to call `.transform` here.

Reply:
Fixed